core/src/main/resources/schema/sqlserver/sqlserver-drop-schema.sql

```diff
@@ -1,3 +1,4 @@
 DROP TABLE IF EXISTS event_tag;
 DROP TABLE IF EXISTS event_journal;
 DROP TABLE IF EXISTS snapshot;
+DROP TABLE IF EXISTS durable_state;
```

Comment:
global_offset?

Reply:
That should be fine, the event_journal's SEQUENCE  doesn't have drop SQL either.